x-pack/test/security_solution_cypress/cypress/e2e/investigations/sourcerer/create_runtime_field.cy.ts

```diff
@@ -48,9 +50,9 @@ describe.skip('Create DataView runtime field', { tags: ['@ess', '@serverless'] }
     visitWithTimeRange(hostsUrl('allHosts'));
     openTimelineUsingToggle();
     populateTimeline();
-    openTimelineFieldsBrowser();
-
-    createField(timelineRuntimeField);
-    cy.get(GET_TIMELINE_HEADER(timelineRuntimeField)).should('exist');
+    createRuntimeFieldFromTimeline(timelineRuntimeField);
```

Comment:
Sweet, will pull in your changes here: https://github.com/elastic/kibana/pull/195662/files#diff-57ff085e8c0983eb6cbfb4fa9565a8a72a77f6a6a7a230eb96cf64f17feb1166 😄